src/main/kotlin/com/hibob/academy/employeeFeedback/service/feedback/FeedbackFetcher.kt

```diff
@@ -0,0 +1,26 @@
+package com.hibob.academy.employeeFeedback.service.feedback
+
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import com.hibob.academy.employeeFeedback.model.Feedback
+import com.hibob.academy.employeeFeedback.model.FilterFeedbackRequest
+import com.hibob.academy.employeeFeedback.model.LoggedInUser
+import com.hibob.academy.employeeFeedback.model.StatusResponse
+import org.springframework.stereotype.Service
+
+@Service
+class FeedbackFetcher(
+    private val feedbackDao: FeedbackDao
+) {
+
+    fun getFeedbackById(loggedInUser: LoggedInUser, feedbackId: Long) : Feedback {
+        return feedbackDao.getFeedbackById(loggedInUser.companyId, feedbackId)
+    }
+
+    fun getFeedbacks(loggedInUser: LoggedInUser, filters: FilterFeedbackRequest) : List<Feedback> {
```

Comment:
don't we want to be able to fetch also by userId?

Reply:
maybe, but it wasn't in the specs